Bundle/ConfigBundle/Controller/GlobalController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Victoire\Bundle\ConfigBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Victoire\Bundle\ConfigBundle\Entity\GlobalConfig;
+use Victoire\Bundle\ConfigBundle\Favicon\FaviconGenerator;
+use Victoire\Bundle\ConfigBundle\Form\GlobalConfigType;
+use Victoire\Bundle\CoreBundle\Controller\VictoireAlertifyControllerTrait;
+
+/**
+ * Global config controller.
+ *
+ * @Route("/victoire-dcms/config/global")
+ */
+class GlobalController extends Controller
+{
+    use VictoireAlertifyControllerTrait;
+
+    /**
+     * Method used to edit global config.
+     *
+     * @param Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     *
+     * @Route("/", name="victoire_config_global_edit")
+     */
+    public function editAction(Request $request)
+    {
+        $entityManager = $this->getDoctrine()->getManager();
+        $globalConfig = $entityManager->getRepository(GlobalConfig::class)->find(1) ?: new GlobalConfig();
```

Comment:
Nothing guarentee that config would keep 1 as ID (therefore if we switch to UUIDs). 
An alternative aproach would be to find the last inserted GlobalConfig according to the updated_at field.